de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringNonHostAddsProjectTest.java

```diff
@@ -1,99 +0,0 @@
-package de.fu_berlin.inf.dpp.stf.test.consistency;
```

Comment:
why do you remove this test?

Reply:
I think something went wrong during moving the files. I found a empty EditDuringNonHostAddsProjectTest.java at the old location. I will restore the file and move it to the new location